sentences/pt-br/_common.yaml

```diff
@@ -364,6 +364,8 @@ expansion_rules:
   falta: "(falta|resta)"
   alarme: "(alarme[s]|cronômetro[s]|timer[s]|temporizador[es])"
   pausar: "(pausa[r]|pause)"
+  cortina: "(cortina[s]|curtinha[s]|persiana[s])"
```

Comment:
I would remove the article 'as', which is already covered via 'a[s]'.
Do you think it is feasible to add 'das' and 'dos'?